pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -24,6 +24,7 @@
 
 TRACER = trace.get_tracer(__name__)
 GEN_PAYLOAD_ARTIFACTS_OUT_DIR = 'gen-payload-artifacts'
+GITHUB_OWNER = "openshift-eng"
```

Comment:
Already defined [here](https://github.com/openshift-eng/art-tools/blob/main/pyartcd/pyartcd/pipelines/build_microshift.py#L431). Can we move it to a common place? e.g. https://github.com/openshift-eng/art-tools/blob/main/pyartcd/pyartcd/constants.py

Reply:
Thanks! Updated